src/cmd/setup-tachyon.js

```diff
@@ -519,13 +552,16 @@ Welcome to the Particle Tachyon setup! This interactive command:
 		].join(os.EOL);
 
 		// Create a temporary file for the XML content
-		const tempFile = temp.openSync();
+		const tempFile = temp.openSync({ prefix: 'config', suffix: '.xml' });
 		fs.writeSync(tempFile.fd, xmlContent, 0, xmlContent.length, 0);
 		fs.closeSync(tempFile.fd);
 		return tempFile.path;
 	}
 
-	async _flash(files) {
+	async _flash({ files, skipFlashingOs, output }) {
+
+		const packagePath = files[0];
+
 		const question = {
```

Comment:
Will this question be asked if `--load_config` is passed? Remember the goal of load config is to have a setup without a single question. Andrew will use this in the Tachyon CI.

Reply:
This will still ask the question. Let me fix it.

Reply:
I removed this. There is another question about product. Is that OK? or should we get that info from the config file as well?